src/jaxoplanet/experimental/starry/utils.py

```diff
@@ -0,0 +1,91 @@
+import jax.numpy as jnp
+import numpy as np
+from scipy.spatial.transform import Rotation
+
+from jaxoplanet.experimental.starry.basis import A1, poly_basis
+from jaxoplanet.experimental.starry.rotation import left_project
+
+
+def ortho_grid(res):
```

Comment:
I think that `res` needs to be an `int` and it needs to be static at compile time right. Perhaps we should wrap this in

```python
@partial(jax.jit, static_argnums=0)
```